Cargo.toml

```diff
@@ -2,14 +2,15 @@
 name = "numpy"
 version = "0.15.0"
 authors = [
-    "Toshiki Teramura <toshiki.teramura@gmail.com>",
-    "Yuji Kanagawa <yuji.kngw.80s.revive@gmail.com>",
+    "The rust-numpy developers",
+    "PyO3 Project and Contributors <https://github.com/PyO3>"
 ]
 description = "Rust binding of NumPy C-API"
```

Comment:
I think this is customarily called "Rust bindings" (plural)?

Reply:
Thanks!